notebooks/problem-sets/Quantum states and operations/02.Single_qubit_quantum_gate.ipynb

```diff
@@ -0,0 +1,72 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d7613bc7",
+   "metadata": {},
+   "source": [
+    "## Problem 2\n",
+    "\n",
+    "<!-- ::: q-quiz(goal=\"test-problem\") -->\n",
```

Comment:
the `goal` should be a unique value. it appears you are using `test-problem` multiple times